Makefile

```diff
@@ -4,5 +4,21 @@ install:
 serve:
 	go run cmd/serve/main.go
 
-deps: 
-	go get ./...
\ No newline at end of file
+build:
```

Comment:
Can you copy over the version and docker build commands to the Makefile? Can also add the `$(shell whoami)` for building custom dev images. We should also setup a docker registry, could just use dockerhub or even the new github packages stuff maybe?

Reply:
Yeah I can of course I'll open another PR with it, I've created a bamdockerhub user that we can all use! I'll throw the pw in the group chat! 

Cheers for the review ✊

Reply:
Update here => https://github.com/blockchain-abstraction-middleware/project-template/pull/3/files#diff-b67911656ef5d18c4ae36cb6741b7965R9